### PR TITLE
Standard labels

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description
This PR adds code that updates the standard labels used across all of CC's applications. Existing labels will not be affected. You can use the new labels in new issues for now, old issues will be automatically moved later.

Oh and it adds a new logging framework that
- greatly reduces the need for manual indentation with spaces in `print()` statements 
- prints beautiful, colourful logs

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: normalize_repos/labels.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
```
Comment (kgodey) [inline normalize_repos/labels.json:9]: Please capitalize ASAP. Also, please change to "Issue must be fixed ASAP", it isn't always related to production breaking.

File: normalize_repos/labels.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83D\uDFE5"
+        },
+        {
+          "name": "high",
+          "color": "NEGATIVE",
+          "description": "Issue stalls works on the project or its dependents",
```
Comment (kgodey) [inline normalize_repos/labels.json:15]: "stalls works" --> "stalls work"

File: normalize_repos/labels.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83D\uDFE7"
+        },
+        {
+          "name": "medium",
+          "color": "NEUTRAL",
+          "description": "Issue is not up actively harmful but that can escalate",
```
Comment (kgodey) [inline normalize_repos/labels.json:21]: I'm not sure what this description means. Maybe something like "Issue is not blocking but should be fixed soon."?

Comment (kgodey) [reply normalize_repos/labels.json:21]: Issues can be for new features, which doesn't have anything to do with harming users by their absence.

File: normalize_repos/labels.json [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83D\uDFE8"
+        },
+        {
+          "name": "low",
+          "color": "POSITIVE",
+          "description": "Issue doesn't hurt anyone",
```
Comment (kgodey) [inline normalize_repos/labels.json:27]: Issues can be for new features, which doesn't have anything to do with harming users by their absence. I would just say that there's no hurry in resolving this issue.

File: normalize_repos/labels.json [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+        }
+      ]
+    },
+    {
+      "name": "status",
+      "labels": [
+        {
+          "name": "ready for dev",
+          "color": "LIGHT",
+          "description": "Issue is ready for work as blockers have been resolved",
```
Comment (kgodey) [inline normalize_repos/labels.json:38]: I'd remove "as blockers have been resolved", there might have not been any blockers to resolve.

File: normalize_repos/labels.json [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83C\uDFC1"
+        },
+        {
+          "name": "blocked",
+          "color": "MEDIUM",
+          "description": "Issue is not ready for work due to a PR or issue blocking it",
```
Comment (kgodey) [inline normalize_repos/labels.json:44]: Blockers may not just be a PR or issue; e.g. we may need to coordinate with a third-party. It's probably best not to specify what the blocker is.

File: normalize_repos/labels.json [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83D\uDEA7"
+        },
+        {
+          "name": "ticket work required",
+          "color": "DARK",
+          "description": "Issue needs details and expansion before it can be worked on",
```
Comment (kgodey) [inline normalize_repos/labels.json:50]: "details and expansion" --> "more details added"

File: normalize_repos/labels.json [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83E\uDDF9"
+        },
+        {
+          "name": "discarded",
+          "color": "LIGHTER",
+          "description": "Issue will not be addressed or worked on",
```
Comment (kgodey) [inline normalize_repos/labels.json:56]: "Issue will not be worked on"

File: normalize_repos/labels.json [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+        }
+      ]
+    },
+    {
+      "name": "goal",
+      "color": "ffffff",
+      "labels": [
+        {
+          "name": "addition",
+          "description": "A new feature or component for the application",
```
Comment (kgodey) [inline normalize_repos/labels.json:73]: "A new feature" seems short and simple

File: normalize_repos/labels.json [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "\uD83C\uDF1F"
+        },
+        {
+          "name": "improvement",
+          "description": "Changes to an existing feature or component in the application",
```
Comment (kgodey) [inline normalize_repos/labels.json:78]: "Changes to an existing feature" seems simpler

File: normalize_repos/labels.json [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+          "emoji": "✨"
+        },
+        {
+          "name": "fix",
+          "description": "Correction in a misbehaving feature or component of the application",
```
Comment (kgodey) [inline normalize_repos/labels.json:83]: "Bug fix" ?

File: normalize_repos/skills.json [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+    "Python"
+  ],
+  "vocabulary": [
+    "Sass",
+    "JavaScript/Storybook"
+  ],
+  "vue-vocabulary": [
+    "JavaScript/Vue",
+    "JavaScript/Storybook"
+  ],
```
Comment (zackkrida) [inline normalize_repos/skills.json:12]: RIP `vue-vocabulary`. This does make me wonder—how does this script react to a missing or archived repository? As long as it doesn't block or end the script prematurely I don't think it is a big issue.

**Review by zackkrida — APPROVED** [review]
This is awesome! Love the addition of the logger.

**Review by kgodey — CHANGES_REQUESTED** [review]
@dhruvkb really great work, and I like the new logging system. :)

I hadn't reviewed the label descriptions earlier, I added a bunch of comments to them. Basically, we want the shortest, simplest, label descriptions (that are accurate) so that they're both quick/easy to read and more accessible to non-native English speakers. Please feel free to make other changes as well, I only commented on some of them, and even those may be able to be simplified.

File: normalize_repos/labels.json (hunk 11 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 10 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 9 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 8 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 7 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 6 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 5 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 3 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 2 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/labels.json (hunk 1 above)
Comment (dhruvkb) [reply normalize_repos/labels.json]: Updated in caf892e.

File: normalize_repos/skills.json (hunk 12 above)
Comment (dhruvkb) [reply normalize_repos/skills.json]: Nevertheless, fixed in 2331482..

**Review by kgodey — APPROVED** [review]

**Pull request merged**